### DISCUSSION TITLE
Welcome to territorial_impact_platform Discussions!

### DISCUSSION THREAD

**Discussion by endorama** [dedelabs/territorial_impact_platform#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mukkoo** [comment]
Each discussion includes a user story based on Miro board.
https://www.dropbox.com/s/zlp1rlzeke3dw66/DEDE%20Entities.jpg?dl=0

Inside each discussion there is tasks and open questions to complete story.
After discussion we will convert tasks to gh issues.

Discussions have a tag corresponding to related persona.
